guides/source/debugging_rails_applications.md

```diff
@@ -458,12 +459,12 @@ The debugger can list, stop, resume and switch between running threads by using
 the `thread` command (or the abbreviated `th`). This command has a handful of
 options:
 
-* `thread` shows the current thread.
-* `thread list` is used to list all threads and their statuses. The plus +
+* `thread`: shows the current thread.
+* `thread list`: is used to list all threads and their statuses. The plus +
 character and the number indicates the current thread of execution.
-* `thread stop _n_` stop thread _n_.
-* `thread resume _n_` resumes thread _n_.
-* `thread switch _n_` switches the current thread context to _n_.
+* `thread stop _n_`: stop thread _n_.
+* `thread resume _n_`: resumes thread _n_.
+* `thread switch _n_`: switches the current thread context to _n_.
```

Comment:
Is there a precedent set elsewhere that we add the colon? If not I would call this a cosmetic change which we don't accept because it makes tracking down the history more difficult. Details about what we consider a cosmetic can be found here: http://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#write-your-code